bindings/pyroot_experimental/PyROOT/python/ROOT/_facade.py

```diff
@@ -44,6 +44,26 @@ def __init__(self, module):
         self.__class__.__getattr__ = self._getattr
         self.__class__.__setattr__ = self._setattr
 
+        # Setup import hook
+        self._set_import_hook()
+
+    def _set_import_hook(self):
+        # Set import hook to be able to trigger auto-loading as appropriate
```

Comment:
This comment is what was there already in the old PyROOT. Can we elaborate a bit more on what this import hook does? Perhaps with an example.